### PR TITLE
Update 1.11.4

### PR DESCRIPTION
**Pull request by YummYume**
#77 
Technically the only error remaining is COUNTRY_FLAG_SMALL_TEX_MAX_SIZE but it can wait (see russia2_defines.lua).

### CODE REVIEW

File: common/characters/SOV.txt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+			visible = {
+				always = no
+			}
+			traits = {
+				ataman_transbaikal_host
+			}
+		}
+	}
+
+	SOV_iosif_stalin = { #TODO ?
```
Comment (YummYume) [inline common/characters/SOV.txt:36]: remove TODO

File: common/scripted_triggers/russia2_diplomacy_scripted_trigger.txt [hunk 2]
```diff
@@ -319,7 +319,7 @@ DIPLOMACY_GENERATE_WARGOAL_ENABLE_TRIGGER = {
 			tag = SOV
 			has_government = communism
 			has_completed_focus = SOV_great_purge2
-			has_idea = SOV_Great_purge
+			has_idea = SOV_Great_purge_idea_r2
```
Comment (YummYume) [inline common/scripted_triggers/russia2_diplomacy_scripted_trigger.txt:322]:
```suggestion
			has_idea = SOV_Great_purge_idea_russia2
```

File: common/scripted_effects/russia_reworked_console.txt [hunk 3]
```diff
@@ -312,21 +274,154 @@ d_russia_reworked_search_errors = {
...
+    log = "Russia Reworked : Scanning Character files"
+    if = {
+        limit = {
+            NOT = {
+                SOV = {
+                    has_character = check_advisor_russian_state_russia2
+                }
+            }
+        }
+        log = " Russia2_russian_states.txt Character file seems to be missing"
```
Comment (YummYume) [inline common/scripted_effects/russia_reworked_console.txt:345]:
```suggestion
        log = " Character file Russia2_russian_states.txt seems to be missing"
```

File: common/scripted_effects/russia_reworked_console.txt [hunk 4]
```diff
@@ -312,21 +274,154 @@ d_russia_reworked_search_errors = {
...
+                SOV = {
+                    has_character = check_advisor_russia2_ukr_russia2
+                }
+            }
+        }
+        log = " Russia2_UKR.txt Character file seems to be missing"
+        add_to_variable = { global.russia_reworked_core_file_missing = 1 }
+    }
+
+    SOV = {
```
Comment (YummYume) [inline common/scripted_effects/russia_reworked_console.txt:409]: Don't retire them? (to allow multiple scans)

File: common/scripted_effects/russia_reworked_console.txt [hunk 5]
```diff
@@ -312,21 +274,154 @@ d_russia_reworked_search_errors = {
...
+        }
+    }
+
+    #Characters
+    log = "Russia Reworked : Scanning Character files"
+    if = {
+        limit = {
+            NOT = {
+                SOV = {
+                    has_character = check_advisor_russian_state_russia2
```
Comment (YummYume) [inline common/scripted_effects/russia_reworked_console.txt:341]:
```suggestion
                    has_character = russia_reworked_load_russian_states_characters
```
Rename all of them

File: localisation/english/russia2_localisation_l_english.yml [hunk 6]
```diff
@@ -1145,9 +1150,9 @@
  SOV_spread_revolution_both_desc:0 ""
  SOV_form_commune_eurasia:0 "Form the Commune of Eurasia"
  SOV_form_commune_eurasia_desc:0 ""
- require_every_european_asian_states:0 "Every European and Asian States is owned and controlled by us"
- require_every_european_states:0 "Every European States is owned and controlled by us"
- require_every_asian_states_anarchist:0 "Every Asian States is owned and controlled by us"
+ require_every_european_asian_states:0 "Every §YEuropean and Asian States§! is §Yowned and controlled§! by us"
```
Comment (YummYume) [inline localisation/english/russia2_localisation_l_english.yml]:
```suggestion
 sov_require_every_european_asian_states:0 "§YOwns§! and §YControls§! every §YEuropean§! and §YAsian States§!"
```

File: localisation/english/russia2_localisation_l_english.yml [hunk 7]
```diff
@@ -3501,7 +3514,7 @@
  SOV_fate_of_poland_monarchist_choice_refuse_every_choice:0 "Fate of Poland"
  SOV_fate_of_poland_monarchist_choice_refuse_every_choice_desc:0 ""
  unlock_decision_fate_of_poland_monarchist:0 "£decision_icon_small Unlock Decisions : §YFate of Poland§!"
- decide_to_restore_prussia_and_other_kingdom:0 "Has §Ychosen§! to §Ydisunite Germany§!"
+ decide_to_restore_prussia_and_other_kingdom_for_decisions:0 "Has §Ychosen§! to §Ydisunite Germany§!"
```
Comment (YummYume) [inline localisation/english/russia2_localisation_l_english.yml]:
```suggestion
 decide_to_restore_prussia_and_other_kingdom_for_decisions:0 "§YDisunited Germany§!"
```

File: localisation/english/russia2_localisation_l_english.yml [hunk 8]
```diff
@@ -5317,6 +5376,7 @@
  russia2_russia.237.d:0 "The Soviet generals we imprisoned are clearly not in a good position to prepare any coup whatsoever. There is a possibility that they'll join our army if we decide to give them another chance. What should we do?"
  russia2_russia.237.a:0 "We need them."
  russia2_russia.237.b:0 "We will not give them a second chance."
+ SOV_russia2_general_communist_will_be_freed:0 "§YEvery Communist§! will be §Yreleased from Prison§! and §Gjoin us§! !"
```
Comment (YummYume) [inline localisation/english/russia2_localisation_l_english.yml]:
```suggestion
 SOV_russia2_general_communist_will_be_freed:0 "§YCommunists§! will be §YReleased§! from §YPrison§! and §GJoin Us§!."
```

File: localisation/english/russia2_localisation_l_english.yml [hunk 9]
```diff
@@ -5877,7 +5938,7 @@
  russia2_russia.419.a:0 "They are cornered, no point in more bloodshed"
  russia2_russia.419.b:0 "Traitors are to be executed!"
  soviet_general_will_survive:0 "We will §Yemprisonned§! all the §Ysoviet general§! that are §Ystill alive§!, maybe they §Gcould be useful in the futur§!"
- not_kill_soviet_general:0 "Emprisonned the §YSoviet Generals§!"
+ not_kill_soviet_general_anarchist_russia:0 "Emprisonned the §YSoviet Generals§!"
```
Comment (YummYume) [inline localisation/english/russia2_localisation_l_english.yml]:
```suggestion
 not_kill_soviet_general_anarchist_russia:0 "§YImprisoned§! the §YSoviet Generals§!"
```

File: localisation/braz_por/russia2_localisation_l_braz_por.yml [hunk 10]
```diff
@@ -901,7 +901,7 @@
  SOV_core_baltic_state_desc:0 "[FROM.GetName] is part of our realm. We have to remind the people of the old days to earn their trust again."
  SOV_unhappy_peasants:0 "Unhappy Peasants"
  SOV_unhappy_peasants_desc:0 "The industrialization of the country required some sacrifices to be made, such as taxing the Peasants and taking their land away. The Kulaks, in particular, have been the target of the Five Year Plan since the start, aiming to reduce their number to none and force them to work in the cities. We have to keep an eye on them."
- SOV_peasants_rebelion:0 "The Peasants Strikes"
+ SOV_peasants_rebelion:0 "The Peasants' Strikes"
```
Comment (Raider472) [inline localisation/braz_por/russia2_localisation_l_braz_por.yml:904]: SOV_peasants_rebelion:0 "The Peasants Strikes"
 SOV_peasants_rebelion_desc:0 "The Peasant's Strikes"

File: localisation/english/russia2_localisation_l_english.yml [hunk 11]
```diff
@@ -4752,7 +4758,7 @@
  russia2_russia.15.t:0 "Our Great Return"
  russia2_russia.15.d:0 "It has finally happened. Unthinkable a few months back, we now control our original key cities and can call ourselves Russia again.\n\nWe shall never let the Germans or anyone else take our precious territory again, and we shall rebuild the country as new. For the Motherland!"
  russia2_russia.15.a:0 "The good ending for us."
- russia2_russia.18.t:0 "The Peasants Strikes"
+ russia2_russia.18.t:0 "The Peasants' Strikes"
```
Comment (Raider472) [inline localisation/english/russia2_localisation_l_english.yml:4761]: russia2_russia.18.t:0 "The Peasant's Strikes"

**Review by Raider472 — APPROVED** [review]

**Pull request merged**